backend/src/main/java/codezap/global/auditing/SkipModifiedAtUpdateBaseTimeEntity.java

```diff
@@ -0,0 +1,43 @@
+package codezap.global.auditing;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.MappedSuperclass;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.PostPersist;
+import jakarta.persistence.PostUpdate;
+import jakarta.persistence.PreUpdate;
+import jakarta.persistence.Transient;
+
+import lombok.Getter;
+
+@Getter
+@MappedSuperclass
+public abstract class SkipModifiedAtUpdateBaseTimeEntity extends BaseTimeEntity {
+
+    @Transient
+    protected LocalDateTime lastKnownModifiedAt;
+
+    @Transient
+    protected boolean isModified = true;
+
+    @PreUpdate
+    private void preUpdate() {
+        if (!isModified && lastKnownModifiedAt != null) {
+            modifiedAt = lastKnownModifiedAt;
+            return;
+        }
+        isModified = true;
+    }
+
+    @PostLoad
+    @PostPersist
+    @PostUpdate
+    private void postLoad() {
+        lastKnownModifiedAt = modifiedAt;
+    }
+
+    public void markUnModified() {
```

Comment:
번뜩 생각나서 올려요

```suggestion
    public void skipModifiedAtUpdate() {
```

클래스명과 맞추기 위해 메서드명 이렇게 변경 제안합니다